examples/lox/src/language/type-system/lox-type-checking.ts

```diff
@@ -47,44 +60,6 @@ export function createTypir(domainNodeEntry: AstNode): Typir {
         inferenceRules: isNilLiteral }); // TODO for what is this used?
     const typeAny = anyKind.createTopType({});
 
-    // utility function to map language types to Typir types
-    // TODO get rid of these type dispatch!
```

Comment:
I am happy if this is not necessary any more

Reply:
Yes, that was quite annoying!